tests/js/spec/components/events/interfaces/spans/waterfallModel.spec.tsx

```diff
@@ -160,4 +160,26 @@ describe('WaterfallModel', () => {
     waterfallModel.toggleAllOperationNameFilters();
     expect(waterfallModel.operationNameFilters).toEqual(noFilter);
   });
+
+  it('querySpanSearch', async () => {
+    const waterfallModel = new WaterfallModel(event);
+    expect(waterfallModel.fuse).toBe(undefined);
+
+    // Fuzzy search needs to be loaded asynchronously
+    // @ts-expect-error
```

Comment:
is this `@ts-expect-error` necessary here?

Reply:
@Zylphrex  Yep, it's necessary since typescript doesn't understand the `tick()` global function.

Other tests written in typescript have this pattern. So I copied it here. e.g. https://github.com/getsentry/sentry/blob/5dd3fb1eb267cdca5881c7cce68a061f21e37ba6/tests/js/spec/views/releases/utils/releaseHealthRequest.spec.tsx#L203

Having this ts comment directive isn't ideal, so I'll raise this in a frontend tsc meeting to see if there's a better way moving forward.